prov/verbs/src/ep_rdm/verbs_utils.h

```diff
@@ -45,6 +45,20 @@
 #include <rdma/fi_log.h>
 #include "../fi_verbs.h"
 
+#if defined(__ICC) || defined(__INTEL_COMPILER) || \
+ defined(__GNUC__) || defined(__GNUG__)
+#include "xmmintrin.h"
+#endif /* ICC || GCC */
+
+#if defined(__ICC) || defined(__INTEL_COMPILER) || \
+ defined(__GNUC__) || defined(__GNUG__)
+#define FI_IBV_PREFETCH_ADDR(_addr) {                    \
```

Comment:
Move the include from above here to avoid listing the same set of defines twice in a row.